src/sentry/utils/event.py

```diff
@@ -90,3 +91,22 @@ def is_event_from_browser_javascript_sdk(event: dict[str, Any]) -> bool:
         "sentry.javascript.svelte",
         "sentry.javascript.sveltekit",
     ]
+
+
+def track_event(
```

Comment:
```suggestion
def track_event_since_received(
```

nit: I realize I suggested this originally but seeing it used feels a bit confusing since "track" and "event" have so many meanings